### DISCUSSION TITLE
Do I need to configure something to make it work?

### DISCUSSION THREAD

**Discussion by bbuhler** [EnzDev/idea-ngx-translate-autocomplete#44] [Q&A] [answered]
Hello (:

I am happy I found your plug-in.
Unfortunately I doesn't seem to work. All translation keys are shown in red.

![image](https://user-images.githubusercontent.com/2802675/161701749-e3d14de6-baf3-448f-9698-a2eedb895996.png)

So I was wondering if I need to configure something.

The translation files are under `src/assets/i18n/`. The JSON consists of nested properties.

**Comment by EnzDev** [comment, marked as answer]
You can configure the path of your translations in IntelliJ Settings (`ctrl + maj + s`) > `Tools` > `NgTranslate Toolset`.
Nested properties are supported :)

**Reply by bbuhler** [reply]
Thank you! Haven't found it before in settings for some reason.
